hawc/apps/common/table.py

```diff
@@ -0,0 +1,60 @@
+from html.parser import HTMLParser
+from typing import List
+
+from docx import Document
+from pydantic import BaseModel, conint, validator
+
+
+class GenericTableCell(BaseModel):
+    row: conint(ge=0)
+    column: conint(ge=0)
+    html_text: str
+
+    def to_docx(self, par):
+        parser = CellHTMLParser()
+        return parser.feed(self.html_text, par)
+
+
+class GenericTable(BaseModel):
+    rows: conint(gt=0)
+    columns: conint(gt=0)
+    cells: List[GenericTableCell]
```

Comment:
we may also want to add `first_row_header: bool = True` and `second_row_header:bool = False`; we could also add `header_cells: List[GenericTableCell]` to make it more generic, but i'd say 99.99% of tables i see have one or two top header rows.